templates/search_results.hbs

```diff
@@ -5,103 +5,102 @@
+        </section>
+      {{/if}}
+      {{#if help_center.community_enabled}}
+        <section class="filters-in-section collapsible-sidebar">
+          <h3 class="collapsible-sidebar-title sidenav-title">{{t 'filter_by_type'}}</h3>
+          <ul>
+            {{#each filters}}
+              <li>
+                <a href="{{url}}" class="sidenav-item" aria-selected="{{selected}}">{{name}} ({{count}})</a>
+              </li>
+            {{/each}}
+          </ul>
+        </section>
+      {{/if}}
+      {{#if subfilters}}
+        <section class="filters-in-section collapsible-sidebar">
+          {{#is current_filter.identifier 'knowledge_base'}}
+            <h3 class="collapsible-sidebar-title sidenav-title">{{t 'filter_by_category'}}</h3>
+          {{/is}}
+          {{#is current_filter.identifier 'community'}}
+            <h3 class="collapsible-sidebar-title sidenav-title">{{t 'filter_by_topic'}}</h3>
+          {{/is}}
+          <ul>
+            {{#each subfilters}}
+              <li>
+                <a href="{{url}}" class="sidenav-item" aria-selected="{{selected}}">{{name}} ({{count}})</a>
+              </li>
+            {{/each}}
+          </ul>
+        </section>
+      {{/if}}
+    </section>
+
     <section class="search-results-column">
-      <h2 class="search-results-subheading">
-        {{t 'knowledge_base'}}
-      </h2>
-      {{#if article_results}}
+      <h3 class="results-subheading">
+        {{#is current_filter.identifier 'unified'}}
+          {{t 'results' query=query count=results_count}}
+        {{else}}
+          {{#unless current_subfilter.identifier}}
+            {{t 'results' query=query count=results_count}}
+          {{else}}
+            {{t 'results_with_scope' query=query count=results_count scope_name=current_subfilter.name}}
+          {{/unless}}
+        {{/is}}
+      </h3>
+      {{#if results}}
         <ul class="search-results-list">
-          {{#each article_results}}
-            <li class="search-result">
-              <a href="{{url}}" class="search-result-link">{{title}}</a>
-              {{#if vote_sum}}
-                <span class="search-result-votes meta-count">{{vote_sum}}</span>
-              {{/if}}
+          {{#each results}}
+            <li class="search-results-list-item result-{{type}}">
+              <a href="{{url}}" class="results-list-item-link">{{title}}</a>
+              <div class="search-results-icons">
+                {{#if vote_sum}}
+                  <span class="search-results-votes">{{vote_sum}}</span>
+                {{/if}}
+                {{#if comment_count}}
+                  <span class="search-results-meta-count">
+                    {{comment_count}}
+                  </span>
+                {{/if}}
+              </div>
               <ul class="meta-group">
                 <li>
-                  <ol class="breadcrumbs search-result-breadcrumbs">
+                  <ol class="breadcrumbs search-results-breadcrumbs">
```

Comment:
These class renames break the styling here https://github.com/zendesk/copenhagen_theme/blob/bc646972db322ba09bd877602ef36465d51eaed1/styles/_search_results.scss#L100